cpt/run_in_docker.py

```diff
@@ -16,8 +16,6 @@ def run():
     # Get all from environ
     conan_api, client_cache, _ = Conan.factory()
     printer = Printer()
-    if os.path.exists(client_cache.default_profile_path):
```

Comment:
This could be a problem when people force sharing the cache between the host and the container, but by the way, that's a bad practice, so I think I'm ok. Let's see how bad it could affect... :hamburger: